packages/react-composites/src/composites/CallComposite/pages/DtmfDialpadPage.tsx

```diff
@@ -61,7 +61,7 @@ const DtmfDialpadPageContent = (props: DialpadPageContentProps): JSX.Element =>
     <Stack style={{ height: '100%', width: '100%', background: theme.palette.white }}>
       <Stack verticalAlign={'center'} style={{ margin: 'auto' }}>
         <DtmfDialerContentTimer />
-        <Text style={{ margin: 'auto' }}>{calleeName !== 'Unnamed participant' ? calleeName : ''}</Text>
+        <Text style={{ margin: 'auto' }}>{calleeName !== 'Unnamed participant' ? calleeName?.toString() : ''}</Text>
```

Comment:
Will this display `undefined` as a string in the DOM if calleeName is undefined ? (the way javascript auto converts types)

To be sure, consider instead having:
`calleeName && calleeName !== 'Unnamed participant' ? calleeName.toString() : ''`